tests/subsys/net/lib/download_client/testcase.yaml

```diff
@@ -0,0 +1,7 @@
+tests:
+  net.lib.download_client:
+    tags: fota
+    platform_allow: nrf9160dk_nrf9160 nrf9160dk_nrf9160_ns qemu_cortex_m3
+    integration_platforms:
+      - nrf9160dk_nrf9160
+      - nrf9160dk_nrf9160_ns
```

Comment:
Please change `qemu_cortex_m3` to `native_posix` platform and add it also into `integration_platforms`, like this:
```
platform_allow: native_posix nrf9160dk_nrf9160 nrf9160dk_nrf9160_ns
integration_platforms:
  - native_posix
  - nrf9160dk_nrf9160
  - nrf9160dk_nrf9160_ns
```

Reply:
Thanks, I did the changes as you have suggested